From 8a57af71779e274d31fe7f87a75bc937a59575db Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 27 Apr 2021 16:12:12 +0100
Subject: [PATCH] docs: -o openstack: Clarify name of file containing OpenStack
 auth

In particular, don't use "stackrc" since you will likely be connecting
to the undercloud.

Thanks: Ming Xie
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1871754
(cherry picked from commit 61b4b5cc2f64e7a642ea03681f36829dbe665825)
---
 docs/virt-v2v-output-openstack.pod | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/docs/virt-v2v-output-openstack.pod b/docs/virt-v2v-output-openstack.pod
index 78623d97..f5a3abad 100644
--- a/docs/virt-v2v-output-openstack.pod
+++ b/docs/virt-v2v-output-openstack.pod
@@ -89,8 +89,8 @@ endpoints.  You will need to either set up your C<$OS_*> environment
 variables or use output options on the virt-v2v command line to
 authenticate with OpenStack.
 
-Normally there is a file called something like C<stackrc>,
-C<overcloudrc> etc which you can simply C<source> to set everything up.
+Normally there is a file called C<overcloudrc> or C<keystonerc_admin>
+which you can simply C<source> to set everything up.
 
 For example:
 
@@ -179,8 +179,10 @@ To output to OpenStack Glance, use the I<-o glance> option.
 
 This runs the L<glance(1)> CLI program which must be installed on the
 virt-v2v conversion host.  For authentication to work, you will need
-to set C<OS_*> environment variables.  In most cases you can do this
-by sourcing a file called something like F<keystonerc_admin>.
+to set C<OS_*> environment variables.
+
+Normally there is a file called C<overcloudrc> or C<keystonerc_admin>
+which you can simply C<source> to set everything up.
 
 Virt-v2v adds metadata for the guest to Glance, describing such things
 as the guest operating system and what drivers it requires.  The
